server/openslides/users/views.py

```diff
@@ -150,6 +150,10 @@ def update(self, request, *args, **kwargs):
             if str(request.user.pk) != self.kwargs["pk"]:
                 self.permission_denied(request)
 
+            # Only admins can set vote delegation
```

Comment:
This should not be necessary. See some lines below: All keys, that are not `("username", "about_me", "email")` will be deleted.

Reply:
I thought that it was clearer to raise a 403 if a user tries to update something which he is not allowed to, but I dind't realise that that is currently not the case for the other fields, either. But if I leave it out, the following request would return a 200 without actually updating something:

    client.patch(
        reverse("user-detail", args=[user.pk]),
        {"vote_delegated_to_id": admin_pk},
    )

Is that intended? It seems very misleading to me.

Reply:
Hm. BTW: This is the same argument as norman always says for OS4 ;)

Yes, it is not nice I guess. Maybe - if you are working on it anyways - always raise an error, if fields are given, which are not expected.

Reply:
Note: This has to be checked in the client! (Possibly make explicit request data with only allowed keys)

Reply:
Mmh, I fear that I break something when doing this xD if it's consistent I'd say we leave it as is. It works and we don't want to put too much work into os3 ;)

Reply:
Ok, fair enough.